vagrant/.gitignore

```diff
@@ -1 +1,3 @@
-.vagrant
\ No newline at end of file
+.vagrant
+*.box
+Berksfile.lock
```

Comment:
I just noticed this on the berkshelf website:

```
A Berksfile.lock will also be created. Add this to version control if you want to ensure that other developers (or your build server) will use the same versions of all cookbook dependencies.
```

I don't mean to turn this into a big discussion, but I'm curious what you meant about cookbook version resolution?